### ISSUE TITLE
Adding Phytophthora pistaciae?

### ISSUE THREAD

**Issue by peterjc** [peterjc/thapbi-pict#301]
Possible references:

```
>HM585360.1 Phytophthora pistaciae;HM585362.1 Phytophthora pistaciae;HM585363.1 Phytophthora pistaciae;HM585364.1 Phytophthora pistaciae
TTTCCGTAGGTGAACCTGCGGAAGGATCATTACCACACCTAAAAAACTTTCCACGTGAAC
CGTATCAACAAGTAGTTGGGGGCCTGCTCTGTGTGGCTGGCTGTCGATGTCAAAGTCGGC
GGCTGGCTGCTATGTGGCGGGCTCTATCATGGCGATTGGTTTGGGTCCTCCTCGTGGGGA 
ACTGGATCATGAGCCCACTTTTTAAACCCATTCTTGATTACTGAATATACT
```

https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=174576

Cropped up looking at a public dataset

**Comment by peterjc** [comment]
Currently our curated DB file only has:

```
>AF403505.X Phytophthora pistaciae
tttccgtaggtgaacctgcggaaggatcattaCCACACCTAAAAAACTTTCCACGTGAAC
CGCATCAACAAGTAGTTGGGGGCCTGCTCTGTGTGGCTGGCTGTCGATGTCAAAGTCGGC
GGCTGGCTGCTATGTGGCGGGCTCTATCATGGCGATTGGTTTGGGTCCTCCTCGTGGGGA
ACTGGATCATGAGCCCACTTTTTAAACCCATTCTTGATTACTGAATATACT
```

**Comment by peterjc** [comment]
That's a single SNP at position 63 (third base of second line as shown), ``T`` in HM585360.1 etc, but ``C`` in AF403505 - David says checking the original capillary data for AF403505 would be tricky (over 20 years old now).

**Comment by peterjc** [comment]
Closed by #407.
